android/app/src/main/java/io/peggyjo/peggymeter/HistoryTextFragment.java

```diff
@@ -1,13 +1,20 @@
 package io.peggyjo.peggymeter;
 
+import android.app.AlertDialog;
+import android.content.DialogInterface;
 import android.support.v4.app.Fragment;
```

Comment:
Not related to your change but can you sort the imports alphabetically please?

Reply:
Yeah I can do that.